packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -85,11 +86,15 @@
 +    repository: rancher/banzaicloud-fluentd
 +    tag: v1.11.5-alpine-9
 +
++fluentd:
++  resources: {}
```

Comment:
How does this affect the chart level one ? Which takes precedence: https://github.com/rancher/charts/blob/dev-v2.5-alt/charts/rancher-logging/rancher-logging/3.9.000-rc03/values.yaml#L23

Reply:
The chart level is for the operator install. The fluentbit and fluentd settings are for the Logging CRs in the overlay.

I think this is out of scope for this PR, but we may want to reorganize values.yaml to have an "overlay" section (or something similar) to denote all of our additions.
CC: @paynejacob

Reply:
Ok. So to clarify, the chart level one did not affect fluent d/bit before and still won’t.